src/pages/trade-options/options-table/options-table.tsx

```diff
@@ -90,12 +93,23 @@ export default function OptionsTable(props: TablePropsType): ReactElement{
                                     </p>
                                 </td>
                                 <td id={createId("td5",index,option)} className={greenCell(option)}>
-                                    <p id={createId("td5p1",index,option)}>
-                                        {opt}
-                                    </p>
-                                    <p id={createId("td5p2",index,option)}>
-                                        $ {(opt*btcPrice).toFixed(2)}
-                                    </p>
+                                    
+                                    <div className="row">
+                                        <div className="col-12 table-input">
+                                            <div className="quantity-label">Quantity:</div>
+                                            <div className="quantity-wrapper">
+                                                <input id="quantity-input"name="quanity" type="number"></input>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div className="row">
+                                        <div className="col-6">
+                                            <button className="buy-button" type="submit">Buy</button>
```

Comment:
On the buy and sell button we will need the current price of that action. So on the Buy button would say 123 USDT which means that with the quantity entered in that row, it would cost the trader 123 USDT to buy that option. Same with the Sell button.